.gitignore

```diff
@@ -1 +1,2 @@
+mytest/*
```

Comment:
Hi,

I really understand how frustrating it is and am really sorry. For now unfortunately I don't really have time to look at it deeply (and that would be needed to give you some useful feedback).

I'll sure let you know when I have some time and review your contribution.